platform_packages.md

```diff
@@ -113,7 +113,7 @@ You may need to install Comodo RSA Domain Validation Secure Server Certificate A
 
 Platform | Author |
 ---------| -------|
-[Arch Linux](https://aur.archlinux.org/packages/drive) | [Jonathan Jenkins](https://github.com/shaggytwodope)
```

Comment:
How come @shaggytwodope's reference is being removed? Perhaps he can chime in and also collaboratively make updates?

Reply:
I'm not currently maintaining any AUR packages. So this is perfectly fine to change.

Reply:
Just to add context. The AUR package at https://aur.archlinux.org/packages/drive which maintained by shaggytwodope was removed a few months ago.

Reply:
I see, thank you for the updates both @shaggytwodope and @steakunderscore!